hat/assets/js/apps/Iaso/domains/instances/details.tsx

```diff
@@ -221,6 +223,18 @@ const InstanceDetails: FunctionComponent = () => {
                                     currentInstance={currentInstance}
                                 />
                             </WidgetPaper>
+                            {currentInstance.change_requests && (
```

Comment:
You should check on length, empty arrays are truthy in JS, this will never be false